src/app/page.jsx

```diff
@@ -64,6 +106,16 @@ export default function Home() {
         <EventsSection events={eventsData} />
       </div>
 
+      {/* About Section */}
+      <div id="aboutsection" className="flex flex-col items-center justify-center min-h-screen p-4 pb-10 gap-8 sm:p-100 text-center">
+        <About/>
+      </div>
+
+      {/* About Section */}
```

Comment:
Bad comment

Reply:
Sorry ,i have corrected it now

Reply:
in the lastest commit

Reply:
![Screenshot 2024-11-01 092754](https://github.com/user-attachments/assets/5507c541-de58-4934-9c03-67873de53543)
corrected it now